cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -197,6 +199,13 @@ func getAPIServerCommand(cfg *kubeadmapi.ClusterConfiguration, localAPIEndpoint
 		defaultArguments["feature-gates"] = fmt.Sprintf("%s=%t", features.IPv6DualStack, enabled)
 	}
 
+	// TODO: remove after BoundServiceAccountTokenVolume is GA.
```

Comment:
isn't BoundServiceAccountTokenVolume going GA in 1.20?

Reply:
no, beta... TokenRequestProjection is already beta which allows pods to make use of those volumes themselves. BoundServiceAccountTokenVolume is the feature that switches the default injected volume to be a bound token, and is going from alpha to beta in 1.20